src/reducers/filter.js

```diff
@@ -0,0 +1,26 @@
+function handleCheckFilter(state, groupName, tagName){
+  if(groupName == null || tagName == null) return state;
+
+  groupName = groupName.toLowerCase();
+  tagName = tagName.toLowerCase();
+
+  // Check if state contains the filterItem that was checked
+  // This should always be false, but it's better to be safe than sorry
+  if(!state.hasOwnProperty(groupName) ||
+    !state[groupName].hasOwnProperty(tagName))return state;
+
+  // Create next state
+  const checked = state[groupName][tagName];
+  return {...state, ...{[groupName]: {...state[groupName], [tagName]: !checked}}};
```

Comment:
Tror jeg nevnte det på kommentarsiden, men gjentar det her. Du kan altså enten bruke @arve0 sitt forslag, eller evt skrive det som
```
return {
  ...state
  [groupName]: {
    ...state[groupName],
    [tagName]: !checked
  }
};
```
And remember that while "saving lives" is important, "saving lines" is not. Ikke vær redd for å bruke flere linjer dersom det blir mer oversiktlig :)